packages/core/src/classes/manager/__test__/entity-manager.spec.ts

```diff
@@ -236,9 +259,9 @@ test('will throw when called with a POJO rather than an instance of a entity cla
     addresses: ['address a'],
   };
 
-  await expect(
-    manager.create<User>(userProperties as EntityInstance)
-  ).rejects.toBeInstanceOf(Error);
+  await expect(manager.create<User>(userProperties)).rejects.toBeInstanceOf(
```

Comment:
since there is now a relationship between the type parameter `Entity` and the first parameter, we actually __can't__ keep this test exactly as is.

The options I see are:
- drop the `User` type parameter altogether
- drop the `EntityInstance` cast
- use `EntityInstance` as the type parameter and keep the `EntityInstance` cast on the input
- drop all the type parameters and rely on inference (which will be inferred as the POJO type)

I choose to  drop the cast, as that felt the cleanest to me, in expressing the failure intent (you can't pass POJO objects in when an Entity instance is expected).